src/Simulation/Simulators/ResourcesEstimator/ResourcesEstimator.cs

```diff
@@ -75,10 +75,6 @@ public ResourcesEstimator(QCTraceSimulatorConfiguration config) : base(config)
             {
                 return null;
             }
-            else if (name == MetricsNames.WidthCounter.ExtraWidth)
```

Comment:
What was ExtraWidth in the old tracer? I don't think that was ever documented, but I may be wrong.

Reply:
I will resolve this in favor of the conversation in the beginning of the output, which discusses the name of this metrics. I'm inclining to make it LocalWidth.